packages/adblocker/test/parsing.test.ts

```diff
@@ -160,8 +160,9 @@ describe('Network filters', () => {
     });
 
     it('pprint anchored hostnames', () => {
+      checkToString('||foo.com^', '||foo.com^');
       checkToString('@@||foo.com', '@@||foo.com^');
-      checkToString('@@||foo.com|', '@@||foo.com^|');
+      checkToString('@@||foo.com|', '@@||foo.com^');
```

Comment:
I think the semantics of this is a bit different now. Although I think it might not be 100% correct in the previous case either. I vaguely remember that this case was a little bit hacky.

Reply:
uBlock Origin team replied me with a clarification: `|` is used to match start/end of address.

```
||hostname^|  => http://example.com/ & not http://example.com/blablabla
```

Many thanks to @ ghajini from uBO.